docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -13,20 +13,24 @@ the corresponding command.
 
 ### Centos
 
+```sh
+sudo dnf install perl-libintl
```

Comment:
The changes in the installation instruction also add Locale::MsgFmt. Shouldn't they be added here as well?

This goes for Debian / Ubuntu and FreeBSD below too.

Reply:
Actually it seems that Locale::MsgFmt is not required any more in Engine (and Backend, but I just copied stuff from there) since https://github.com/zonemaster/zonemaster-engine/commit/e8150686afc90ecf18c980226a15e1870b11bdaa so we should remove the dependency in both repositories.

Reply:
You write "it seems that". Does that mean that further tests and investigation is needed or that you have verified that it is not anymore?

Reply:
I got no matches when grepping for Locale::MsgFmt in all our repositories. It should be safe to remove it.

Reply:
Removed in https://github.com/zonemaster/zonemaster-engine/pull/999